README.md

```diff
@@ -64,7 +64,7 @@ npm install
 
 # Rebuild native Node.js modules against the app version of Node.js
 # MacOS, Linux and Git Bash on Windows
-$(npm bin)/electron-rebuild
```

Comment:
This is actually not an error. Paste this command in the terminal and it will work as expected.

Reply:
> This is actually not an error. Paste this command in the terminal and it will work as expected.

I did but it shows me
```
(py27) rituraj@rituraj-G7-7588:~/Documents/git/scancode-workbench$ (npm bin)/electron-rebuild
bash: syntax error near unexpected token `/electron-rebuild'

```

Reply:
You have to use `$(npm bin)` with no spaces between the `$` and the `(npm bin)` parts. It is similar to a bash variable.

Reply:
I got you. I am confused and paste it without $ sign and thought that` (npm bin)/electron-rebuild
` is a command. I think now we should change these lines like-
# Clone this repository
$ git clone https://github.com/nexB/scancode-workbench.git

# Go into the repository
$ cd scancode-workbench

# Install dependencies and run the app
$ npm install

# Rebuild native Node.js modules against the app version of Node.js
# MacOS, Linux and Git Bash on Windows
$ $(npm bin)/electron-rebuild
# Windows except for Git Bash
.\node_modules\.bin\electron-rebuild.cmd

# Run the app
$ npm start